config/initializers/huginn_agent.rb

```diff
@@ -0,0 +1,5 @@
+require 'huginn_agent'
+
+HuginnAgent.hack_huginn_to_accept_me
+
+HuginnAgent.types.each { |t| t.emit }
```

Comment:
I explained in more detail on #920, but this command just loops through all defined ```HuginnAgent```s and creates Agent classes that Huginn will recognize.